astroquery/simbad/core.py

```diff
@@ -300,6 +336,50 @@ class SimbadClass(SimbadBaseQuery):
     def __init__(self):
         super().__init__()
         self._VOTABLE_FIELDS = self._VOTABLE_FIELDS.copy()
+        self._server = conf.server
+        self._tap = None
+
+    @property
+    def server(self):
+        """The Simbad mirror to use."""
+        return self._server
+
+    @server.setter
+    def server(self, server: str):
+        """Allows to switch server between Simbad mirrors.
+
+        Parameters
+        ----------
+        server : str
+            It should be one of `~astroquery.simbad.conf.servers_list`.
+        """
+        if server in conf.servers_list:
+            self._server = server
+        else:
+            raise ValueError(f"'{server}' does not correspond to a Simbad server, "
+                             f"the two existing ones are {conf.servers_list}.")
+
+    @property
+    def tap(self):
+        """A ``~pyvo.dal.tap.TAPService`` service for Simbad."""
```

Comment:
we should be able to intersphinx link to pyvo, the trick is to link to the namespace its API is documented and not to the namespace the source code is
```suggestion
        """A `~pyvo.dal.TAPService` service for Simbad."""
```